compiler/util/src/org/jetbrains/kotlin/config/LanguageVersionSettings.kt

```diff
@@ -285,6 +285,7 @@ enum class LanguageFeature(
     ProhibitAccessToEnumCompanionMembersInEnumConstructorCall(KOTLIN_1_9, kind = BUG_FIX), // KT-49110
     ReferencesToSyntheticJavaProperties(KOTLIN_1_9), // KT-8575
     CustomEqualsInInlineClasses(KOTLIN_1_9), // KT-24874
+    EnhanceNullabilityOfPrimitiveArrays(KOTLIN_1_9, kind = BUG_FIX), // KT-48861
```

Comment:
Minor: normally we refer a LC (language committee) issue here. I created a stub for the issue: KT-54521.

Reply:
done